apps/server/src/modules/authorization/interfaces/index.ts

```diff
@@ -8,6 +8,7 @@ export enum AllowedAuthorizationEntityType {
 	'Task' = 'tasks',
 	'Lesson' = 'lessons',
 	'Team' = 'teams',
+	'Submission' = 'submissions',
 }
 
-export type AllowedEntity = Task | Course | CourseGroup | User | School | Lesson | Team;
+export type AllowedEntity = Task | Course | CourseGroup | User | School | Lesson | Team | Submission;
```

Comment:
It seems like this type is not used anywhere. What is it for?

Reply:
I looked into it yesterday, but still can't really tell. @SevenWaysDP Can you help?